README.rst

```diff
@@ -2,3 +2,36 @@ casepropods.dummy
 ==================
 
 Example casepro data pod returning statically configured data.
+
+Install
+~~~~~~~
+
+.. code-block::
+
+  $ pip install casepropods.dummy
+
+Usage
+~~~~~
+
+In casepro's settings file, add the pod as an installed app, and configure the pods that you would like to show in the UI:
+
+.. code-block:: python
+
+  INSTALLED_APPS += ('casepropods.dummy.plugin.DummyPodPlugin',)
+
+  PODS = [{
+      'label': 'dummy_pod',
+      'title': 'Maternal Health Info',
+      'data': {
+          'items': [{
+              'name': 'EDD',
+              'value': '2015-07-18'
+          }, {
+              'name': 'Clinic Code',
+              'value': '2034 6524 6421'
+          }]
+      }
+  }, {
+      'label': 'custom_pod',
```

Comment:
Hmm. I'm a little worried that people won't know what `custom_pod` is. Should we perhaps leave it out or add a comment that this is a different kind of pod?